cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControlTest.java

```diff
@@ -99,33 +93,29 @@ public void testCCUserTaskDelayVerbose(Vertx vertx, VertxTestContext context) th
     @Test
     public void testMockCCServerPendingCallsOverride(Vertx vertx, VertxTestContext context) throws IOException, URISyntaxException {
 
-        CruiseControlApi client = new CruiseControlApiImpl(vertx);
-        String userTaskID = MockCruiseControl.REBALANCE_NO_GOALS_RESPONSE_UTID;
-
         int pendingCalls1 = 2;
-        Checkpoint firstPending = context.checkpoint(pendingCalls1);
         int pendingCalls2 = 4;
-        Checkpoint secondPending = context.checkpoint(pendingCalls2);
```

Comment:
How is this checked now when you do not use the async checkpoints?

Reply:
Yep, good point. This was causing a false positive, as all assertions after both checkpoints were flagged sufficiently were ignored, but I'll revisit to keep the previous behaviour and ensure the test can fail correctly :)